modules/nixos/monitoring/alert-rules.nix

```diff
@@ -101,8 +101,13 @@ lib.mapAttrsToList
       description = "{{$labels.host}} is running with load15 > 1 for at least 5 minutes: {{$value}}";
     };
 
+    localhost_reboot = {
```

Comment:
This is to ensure that we notice if the matrix alerts stop working.

This machine will reboot once or twice a week after a flake update and send an alert.